vm/spec_test.go

```diff
@@ -0,0 +1,47 @@
+package vm
+
+import (
+	"testing"
+)
+
+func TestSpecSuccessWithoutExit(t *testing.T) {
+	input := `
+require "spec"
+
+Spec.describe Spec do
+  it "fails and exit with code 1" do
+	expect(1).to eq(1)
+  end
+end
+
+Spec.test
+
+10
+`
+	v := initTestVM()
+	result := v.testEval(t, input, getFilename())
+	verifyExpected(t, 0, result, 10)
+}
+
+func TestSpecFailAndExit(t *testing.T) {
```

Comment:
@saveriomiroddi I wrote specs for checking the test exit when failed, but now it exit the whole VM test suite 😅

Reply:
Hehe, yeah, in fact I did wrote a comment specifically about this :joy: 

I think we need to wait for the `rescue` system to be in place, so that we can (possibly) use something around the lines of throwing and rescuing a `SystemExit` exception.

Reply:
if we're still need to implement `begin` and `rescue`, we gonna wait for a long time cause it's very hard. but if we just need `rescue` for the whole method, that should be a lot quicker.

Reply:
hNeeds to be well thought, since testing the framework itself has even extra complexities. :thinking:.